.changeset/tricky-spoons-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+"@animareflection/ui": minor
+---
+
+Refactor library structure with client component focus
```

Comment:
This is something we can discuss: what are your thoughts on known-breaking changes, such as this one, being prefixed with "**Breaking:**" ? Example: https://github.com/wagmi-dev/wagmi/blob/main/packages/core/CHANGELOG.md#minor-changes-4

Note also we can do this at the release PR level, just bringing this up so we have the decision formed moving forward. Personally, I think it's not strictly necessary, but might be nice. Let me know what you think.

Reply:
Yeah that sgtm! I think it would be nice as well.

Reply:
@Twonarly1 you probably saw this discussion but tagging just in case